pkg/framework/util.go

```diff
@@ -31,10 +31,13 @@ import (
 )
 
 const (
-	defaultAPIVersion      string = "0.1.0"
-	defaultUIDPrefix       string = "e2e-cri-uid"
-	defaultNamespacePrefix string = "e2e-cri-namespace"
-	defaultAttempt         uint32 = 2
+	defaultAPIVersion           string = "0.1.0"
+	defaultUIDPrefix            string = "e2e-cri-uid"
+	defaultNamespacePrefix      string = "e2e-cri-namespace"
+	defaultAttempt              uint32 = 2
+	defaultContainerImage       string = "gcr.io/google_containers/busybox:1.24"
+	defaultStopContainerTimeout int64  = 60
+	defaultExecSyncTimeout      string = "5s"
```

Comment:
Why not make `defaultExecSyncTimeout` as `int64` (same with `defaultStopContainerTimeout `)?